tests/integration_test.py

```diff
@@ -114,6 +118,21 @@ def tearDown(self):
 
         self.client.close()
 
+    def run_container(self, *args, **kwargs):
+        container = self.client.create_container(*args, **kwargs)
+        self.client.start(container)
+        self.tmp_containers.append(container)
```

Comment:
I think these two lines need (123, 124) to be swapped, otherwise container start failures will prevent a container from being removed.